_posts/2018-03-08-reclaiming-disk-space-from-mongodb.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Reclaiming Disk Space from MongoDB
+author: David Davis
+---
+MongoDB can often eat up large amounts of disk space especially in deployments that see heavy
+loads. It's possible to reclaim some of this disk space by using Mongo's repair utility. Before you
+start this process though, **it is highly recommended you back up your MongoDB database**.
```

Comment:
s/it is highly recommended you back up your/ it is highly recommended to back up the